src/content/docs/kubernetes-pixie/k8s-otel/install.mdx

```diff
@@ -0,0 +1,635 @@
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.prometheus.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `prometheus` receiver is enabled for data collection.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.prometheus.scrapeInterval`</td>
+            <td>String</td>
+            <td>Sets the scrape interval for the `prometheus` receiver, determining how frequently metrics are collected.</td>
+            <td>`1m`</td>
+        </tr>
+        <tr>
+            <td>`receivers.k8sEvents.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `k8sEvents` receiver is enabled for collecting Kubernetes events.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.hostmetrics.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `hostmetrics` receiver is enabled for collecting host-level metrics.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.hostmetrics.scrapeInterval`</td>
+            <td>String</td>
+            <td>Sets the scrape interval for the `hostmetrics` receiver.</td>
+            <td>`1m`</td>
+        </tr>
+        <tr>
+            <td>`receivers.kubeletstats.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `kubeletstats` receiver is enabled for collecting metrics from the Kubelet.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.kubeletstats.scrapeInterval`</td>
+            <td>String</td>
+            <td>Sets the scrape interval for the `kubeletstats` receiver.</td>
+            <td>`1m`</td>
+        </tr>
+        <tr>
+            <td>`receivers.filelog.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `filelog` receiver is enabled for collecting logs from files, typically container logs.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`lowDataMode`</td>
+            <td>Boolean</td>
+            <td>Enables a mode that sends only the essential metrics required to illuminate the New Relic Kubernetes user interface. This helps reduce data ingestion volume.</td>
+            <td>`true` or `false`</td>
+        </tr>
+    </tbody>
+</table>
+
+  </Collapser>
+</CollapserGroup>
+
+
+## Alerts [#alerts]
+
+You can install essential alert policies by going through the guided installation flow in Integrations & Agents. This automatically sets up alert policy named **Kubernetes (OpenTelemetry) alert policy** in your New Relic account with multiple alert conditions designed for Kubernetes observability.
```

Comment:
It is missing the 2nd part where we explain that customers needs to customize the alerts 

https://newrelic.atlassian.net/wiki/spaces/KOT/pages/4408737833/K8s+OTel+docs#Alerts